dialog/file.go

```diff
@@ -313,10 +313,11 @@ func (f *fileDialog) setLocation(dir fyne.URI) error {
 		if err != nil {
 			return err
 		}
+
 		if !isDir {
 			return errors.New("location was not a listable URI")
 		}
-		f.breadcrumb.Append(
```

Comment:
This was a merge conflict, requesting @andydotxyz to confirm this was the right thing to do.

Reply:
GH is rendering this in a confusing way in the PR review view. I changed `Append` to `Add` but kept everything else the same.

Reply:
looks good